docs/make.jl

```diff
@@ -0,0 +1,25 @@
+push!(LOAD_PATH,"../src/")
+
+using Documenter
+using MCMCChains
+
+name = "MCMCChains.jl"
+
+makedocs(
+    sitename = name,
+    pages = [
+        "MCMCChains" => "index.md",
+        "Plotting" => [
+            "StatsPlots.jl" => "statsplots.md"
+        ],
+        "API" => [
+            "Chains" => "chains.md",
+            "Diagnostics" => "diagnostics.md",
+            "Posterior statistics" => "stats.md",
+            "Model selection" => "modelstats.md"
+        ]
+    ],
+    format = Documenter.HTML(prettyurls = get(ENV, "CI", nothing) == "true")
+)
+
+deploydocs(repo = "github.com/rikhuijzer/$name.git", devbranch = "master")
```

Comment:
The URL is incorrect and `devbranch` is set to the default value. I think it would also be good to build previews: ```suggestion
deploydocs(;
    repo="github.com/TuringLang/MCMCChains.jl.git", push_preview=true
)
```
We should also add an action that cleans the previews (you can find an example in the Documenter docs and https://github.com/JuliaGaussianProcesses/KernelFunctions.jl/blob/master/.github/workflows/doc_preview_cleanup.yml).